sklearn/naive_bayes.py

```diff
@@ -412,8 +413,12 @@ def _update_class_log_prior(self, class_prior=None):
             self.class_log_prior_ = np.log(class_prior)
         elif self.fit_prior:
             # empirical prior, with sample_weight taken into account
-            self.class_log_prior_ = (np.log(self.class_count_)
-                                     - np.log(self.class_count_.sum()))
+            class_count = np.where(self.class_count_ != 0, 
```

Comment:
I'm replacing the 0s in the matrix with 1s as there was a warning,  "RuntimeWarning: divide by zero encountered in log" shown in l:420 and l:421 (line numbers with reference to this PR).